Issue 3205 : Test and edit Kong Gateway (OSS) Quickstart guide (#3270)
Kong/docs.konghq.com: app/gateway-oss/2.5.x/getting-started/adding-consumers.md

  </ol>
</div>

> In the last section, we learnt how to add plugins to Kong, in this section

Reviewer: ```suggestion
In the last section, you learned how to add plugins to Kong, in this section
```

We favor US spelling in our docs set for consistency. I am also swapping "we" for "you" since you do that later on!

  --data "username=Jason"
```

> You can see a response similar to the one below:

Reviewer: ```suggestion
You should see a response similar to the one below:
```

Changing it back to "should" because that implies that the action was potentially successful.

## 3. Verify that your Consumer credentials are valid

You can now issue the following request to verify that the credentials of
your `Jason` Consumer is valid:

```bash

> $ curl -i -X GET \

Reviewer: ```suggestion
curl -i -X GET \
```

## Next Steps

> Now that you've covered the basics of adding Services, Routes, Consumers and enabling

Reviewer: ```suggestion

```

Remove this line in favor of the next suggestion.

> Plugins, feel free to read more on Kong in one of the following documents:

Reviewer: ```suggestion
Read more about the following related topics:
```
